### DISCUSSION TITLE
help me please

### DISCUSSION THREAD

**Discussion by Minyorstaff** [chromiecraft/chromiecraft#2750] [General]
Hi everyone. I'm new to your server. And to be honest,  I really liked your server. I first wanted to lvl up the paladin blood-elf, but after a while, when my paladin became lvl 18-19, I really wanted to play for the Alliance. I really want to continue playing on your server. But only for the Alliance. Playing with 0 for the Alliance, I'm really sorry for my paladin((

Please move my paladins to the side of the alliance. Please. 

I know you have a service. But this is a paid service. 

I play on your server because you have a very good server. The main thing here is no donation and I like it very much. I don't know, maybe I'm wrong, but for the first time in my life I saw the real best server where there is no donation. You can't buy clothes and things for money and it's so cool. 

I hope there won't be donation until the end 

Please again Move my  21 lvl paladin  to the alliance. Please :( 

I really hope that you will accept my request.

**Comment by 55Honey** [comment]
Hello @Minyorstaff,
Thanks for your question.

Faction Transfer is a service which can be obtained through the shop, like you already mentioned. The currency for it, ChromiePoints, can be obtained in many possible ways. Of course one of them is to donate.

You might want to start an alliance character now, and transfer your Paladin later, once you've obtained some ChromiePoints.

Players can participate in events or PvP to receive free points, or contribute to the Server. See https://www.chromiecraft.com/how-to-contribute/ for a lot of ways to help and receive rewards.

**Reply by Minyorstaff** [reply]
And thank you too)) 

To be honest, I was glad that you answered quickly) And it is very nice when the administrators will quickly answer and help you)) 

I apologize if I make any mistakes. English is not a native language 

The main reason why I play pirates is that they are free)) And here people are more polite than on official servers

I would like to ask a few questions. I will be very happy if you answer all my questions. 

1. Thank you for telling me that with the Chromiepoints i can move my paladin to the Alliance right? But how? I didn't understand how it works? 

2. You said that i can get free points via PvP.  can you please tell me  more about this. How can i get Points via PvP I saw that I have a 50 profile on my profile. And judging by the writing I got them through PVP. The funny thing is  i don't know how i got it))) 

3. there is no way to get a free transfer?  I'm afraid that i can't buy points :(((

**Reply by 55Honey** [reply]
You can buy a faction transfer service for Chromie Points:
https://www.chromiecraft.com/product/change-faction/

Here is an example of our monthly ChromiePoints distribution for PvP:
https://www.chromiecraft.com/november-2021-pvpstats-winners/
